include/wx/generic/gridctrl.h

```diff
@@ -252,15 +308,10 @@ class WXDLLIMPEXP_ADV wxGridCellChoiceRenderer : public wxGridCellStringRenderer
     virtual void SetParameters(const wxString& params) wxOVERRIDE;
 
     virtual wxGridCellRenderer *Clone() const wxOVERRIDE
-    {
-        return new wxGridCellChoiceRenderer(*this);
-    }
+        { return new wxGridCellChoiceRenderer(*this); }
 
 protected:
-    wxGridCellChoiceRenderer(const wxGridCellChoiceRenderer& other)
-        : m_choices(other.m_choices)
-    {
-    }
+    wxString GetString(const wxGrid& grid, int row, int col);
```

Comment:
Not sure why this was put in wxGridCellEnumRenderer. It should be in this class, right?

Reply:
I don't think so, this would make `wxGridCellEnumRenderer` and `wxGridCellChoiceRenderer` basically identical and they're supposed to be different: one shows strings themselves while the other one shows their indices.